src/os/ui/timelinepanel.js

```diff
@@ -233,3 +246,40 @@ os.ui.TimelinePanelCtrl.prototype.adjust = function() {
   os.MapContainer.getInstance().updateSize();
   os.dispatcher.dispatchEvent(os.MapEvent.GL_REPAINT);
 };
+
+
+/**
+ * Panel lock button click.
+ * @export
+ */
+os.ui.TimelinePanelCtrl.prototype.lock = function() {
+  this.onBrushLockButtonUp();
+};
+
+
+/**
+ * Synchs both lock buttons
+ */
+os.ui.TimelinePanelCtrl.prototype.setLock = function() {
+  this['locked'] = this.tlc.getLock();
+  os.ui.apply(this.scope);
+};
+
+
+/**
+ * Panel lock button click.
+ * @param {boolean=} opt_getlock
+ */
+os.ui.TimelinePanelCtrl.prototype.onBrushLockButtonUp = function(opt_getlock) {
```

Comment:
This function is only called by `lock`, so the body can be moved there instead of having two functions.